elfpy/data/db_schema.py

```diff
@@ -47,11 +48,14 @@ class WalletInfo(Base):
     sharePrice: Mapped[Union[float, None]] = mapped_column(Numeric, default=None)
 
 
+@dataclass
```

Comment:
The `Base` class already has a [mixin](https://realpython.com/inheritance-composition-python/#mixing-features-with-mixin-classes) that makes it behave like a dataclass:
```
class Base(MappedAsDataclass, DeclarativeBase):
    """Base class to subclass from to define the schema"""
```

As described in the [docs](https://docs.sqlalchemy.org/en/20/orm/dataclasses.html#:~:text=The%20MappedAsDataclass%20mixin%20may%20be%20applied%20either%20to%20the%20Declarative%20Base%20class%20or%20any%20superclass%2C%20as%20in%20the%20example%20below%3A)

Can you explain why you additionally need the `@dataclass` decorator here? I don't understand what it accomplishes.

Reply:
that's explained in the description of the PR that preceded this (#676)
I built on it for this PR (you can see the first commit is just that PR)
that PR is a required here so tests don't break, as explained in that PR

Reply:
actually I just reviewd that PR, and removed the dataclass decorator doesn't introduce any pyright issues. may have been caused by something else I was doing while building the tests. good catch!